client/network/src/protocol.rs

```diff
@@ -1643,9 +1680,7 @@ impl<B: BlockT, H: ExHashT> NetworkBehaviour for Protocol<B, H> {
 					}
 					Some(Fallback::BlockAnnounce) => {
 						if let Ok(announce) = message::BlockAnnounce::decode(&mut message.as_ref()) {
-							let outcome = self.on_block_announce(peer_id.clone(), announce);
-							self.update_peer_info(&peer_id);
-							outcome
+							self.push_block_announce_validation(peer_id, announce)
```

Comment:
Is it safe to ignore the return type of `push_block_announce_validation`?

Reply:
It is not ignored? It is the return value of this if branch

Reply:
Sorry, missed that.